src/app/test-record/test-delete-reason-modal/test-delete-reason-modal.component.scss

```diff
@@ -0,0 +1,26 @@
+.enter-reason-for-cha {
+  color: #0b0c0c;
```

Comment:
let' move this scss in a separate file that we load in _global.scss.
Also let's see use the variables provided by GDS 
https://design-system.service.gov.uk/styles/